packages/util-utf8/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "@aws-sdk/util-utf8",
+  "version": "3.0.0",
+  "description": "A UTF-8 string <-> UInt8Array converter",
+  "main": "./dist-cjs/index.js",
+  "module": "./dist-es/index.js",
+  "scripts": {
+    "build": "concurrently 'yarn:build:cjs' 'yarn:build:es' 'yarn:build:types'",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:es": "tsc -p tsconfig.es.json",
+    "build:include:deps": "lerna run --scope $npm_package_name --include-dependencies build",
+    "build:types": "tsc -p tsconfig.types.json",
+    "build:types:downlevel": "downlevel-dts dist-types dist-types/ts3.4",
+    "clean": "rimraf ./dist-* && rimraf *.tsbuildinfo",
+    "test": "jest"
+  },
+  "author": {
+    "name": "AWS SDK for JavaScript Team",
+    "url": "https://aws.amazon.com/javascript/"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "tslib": "^2.3.1"
```

The dependency `@aws-sdk/util-buffer-from` used in function `fromUtf8` is missing.
```suggestion
  "dependencies": {
    "@aws-sdk/util-buffer-from": "*",
    "tslib": "^2.3.1"
```

---

fixed